src/OverlayTrigger.js

```diff
@@ -106,6 +127,8 @@ function OverlayTrigger({
   popperConfig = {},
   defaultShow,
   delay: propsDelay,
+  placement,
+  flip = placement === 'auto',
```

Comment:
`placement.indexOf('auto') !== -1` probably?

Reply:
You are correct, totally forgot about auto-start and auto-end... :(